### ISSUE TITLE
Notification Blueprint Generation

### ISSUE THREAD

**Issue by SychO9** [flarum/cli#17]

**Comment by SychO9** [comment]
This is very interesting btw, this is a situation where I feel like the separation of backend/frontend doesn't help, if we're generation a new notification type/blueprint, it would be neat  to do everything from backend class to frontend component.

**Comment by askvortsov1** [comment]
> This is very interesting btw, this is a situation where I feel like the separation of backend/frontend doesn't help, if we're generating a new notification type/blueprint, it would be neat to do everything from backend class to frontend component.

Agreed: we should have steps for each, but one command that does backend + frontend + extender.
